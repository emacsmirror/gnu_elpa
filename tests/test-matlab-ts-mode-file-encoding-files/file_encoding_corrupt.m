function file_encoding_corrupt
% bad char: 
    disp("in file_encoding_corrupt");
end
