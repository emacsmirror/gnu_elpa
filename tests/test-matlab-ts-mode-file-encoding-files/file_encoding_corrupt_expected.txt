Major mode errored with message
Not entering matlab-ts-mode due to non-printable utf8 character "" at point 45

Entered major-mode: fundamental-mode
